roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -334,7 +334,11 @@ def write(self):
         if hasattr(self.yaml_dict, 'fa'):
             self.yaml_dict.fa.set_block_style()
 
-        Yedit._write(self.filename, yaml.dump(self.yaml_dict, Dumper=yaml.RoundTripDumper))
+        # pylint: disable=no-member
+        if hasattr(yaml, 'RoundTripDumper'):
```

Comment:
Currently these modules all import ``ruamel.yaml`` directly and will always have ``yaml.RoundTripDumper``. Will these be updated to ``try/except`` an ``ImportError`` like reqoquery?